evaluation_harness/evaluators.py

```diff
@@ -132,6 +138,13 @@ def __call__(
             match approach:
                 case "exact_match":
                     score *= self.exact_match(ref=value, pred=pred)
+                    if value == "N/A" and score != 1:
```

Comment:
Can you make this a separate case rather than under an exact match? This might be confusing